javascript/src/main/pages/Search/Search.js

```diff
@@ -25,9 +33,18 @@ const Search = () => {
                 },
             });
             console.log(`result=${JSON.stringify(result)}`)
-            return result;
+            if(result.items){
+                return result;
+            }
+
+            addToast(`Error: ${JSON.stringify(result)}`, { appearance: "error" });
+
+            return emptyResults;
         } catch (err) {
+
+            addToast(`Error: ${err}`, { appearance: "error" });
```

Comment:
These lines show how to test for this: 

https://github.com/ucsb-cs156-s21/STARTER-team02/blob/b8cfd987fbdb3aa955216e466c7bf3b82d4b12a2/javascript/src/test/pages/LocationsPage.test.js#L81-L105

Reply:
But it's ok to make that a new PR